### PR TITLE
Added community-supported platforms to the getting started

### PR DESCRIPTION
**Pull request by nigerianlegend**
This  helps distinguish between supported and comunity suported platorms.check out PR #1584 for more information.

### CODE REVIEW

**Comment by siddharthist** [comment]
Close, but you actually want this right below the right above it. Also, those links should be removed from the above TOC.

File: docs/getting_started/index.rst [hunk 1]
```diff
@@ -50,11 +50,28 @@ Here are some guides specific to each of the platforms that Mantl supports:
...
+    softplayer.rst
+    bare-metal.rst
+        
 There are several preparatory steps to provisioning the cloud hosts that are
 common to all providers:
 
 Step 1: Copy .tf file
----------------------
+----------------------
+
```
Comment (siddharthist) [inline docs/getting_started/index.rst]: This now has a lot of extra whitespace, can we remove that?

File: docs/getting_started/index.rst [hunk 2]
```diff
@@ -50,11 +50,28 @@ Here are some guides specific to each of the platforms that Mantl supports:
    vagrant.rst
    triton.rst
 
+Community-supported platforms:
+------------------------------
```
Comment (siddharthist) [inline docs/getting_started/index.rst]: This doesn't need the underline, that will make it a header

File: docs/getting_started/index.rst (hunk 1 above)
Comment (nigerianlegend) [inline docs/getting_started/index.rst]: I don't know I guess @siddharthist

Comment (siddharthist) [inline docs/getting_started/index.rst]: @nigerianlegend Is this still on your radar?

**Comment by sehqlr** [comment]
LGTM

**Pull request merged**